data/net/minecraft/gametest/framework/StructureUtils.mapping

```diff
@@ -74,5 +75,6 @@ CLASS net/minecraft/gametest/framework/StructureUtils
 		ARG 1 pos
 		ARG 2 rotation
 		ARG 4 serverLevel
+		ARG 5 dontPlaceOnSizeMismatch
```

Comment:
Could be `ignoreSizeMismatch` instead?

Reply:
It's the opposite of that. If it's true, it doesn't ignore it. If it's false, it does ignore the mismatch.

Reply:
failOnSizeMismatch?

Reply:
I thought about that too, but it's not really failing. Like some logic does run anyways. idk.

Reply:
In this case, it might be useful to add a `@param` comment for this parameter explaining exactly what this means, as it's clearly not obvious just via name (and I don't think that it will be via the choice of the best possible flag name).

Explanatory comment > bikshed parameter name.

Reply:
Ok, added a comment.